backend/app/tests/api/routes/collections/test_collection_info.py

```
@@ -185,3 +185,39 @@ def test_collection_info_not_found_returns_404(
+    client: TestClient,
+    db: Session,
+    user_api_key_header,
+):
+    """
+    Test that when include_docs=true and include_url=true,
+    the endpoint returns documents with their URLs.
+    """
+    project = get_project(db, "Dalgo")
+    collection = get_collection(db, project)
+
+    document = link_document_to_collection(db, collection)
```

Ruff (0.14.10)

Local variable `document` is assigned to but never used

Remove assignment to unused variable `document`

(F841)